_posts/2017-09-05-life-at-codurance-robert-firek.md

```diff
@@ -0,0 +1,91 @@
+
+For me, the biggest difference is the sense of community. Regardless of the projects that we’re working on, the support and knowledge sharing in the team is much stronger than it was in my previous jobs.
+
+We also share and learn ideas outside of our normal project times as well. We organise our own internal sessions after working hours. Say for example that you’re interested in the functional programming or you need to develop your skills in a specific language. You will almost always find more people interested in the same topic. You form a group, book a space in the office and start to study the topic. It is the great way to learn new things and share your knowledge with the others.
+
+
+<blockquote class="hero bq-text-lg"><p>"For me, it’s the sense of community"</p></blockquote>
+
+We also have our bi-weekly catchups on a Tuesday evening, where we meet together on our catch-up meeting to have fun together. We organise a mini-open conference preceded by lightning talks. This is the space where we discuss anything that we want - nothing is off topic. For example, we recently had a few talks about aeronautics. We also have the chance to socialise with people from different projects (which we cannot meet every day). It is the great opportunity for us to build the team and get know each other.
+
+Every two months we organise a bigger version of our catch-up meetings. It is Codurance Day. We spend the whole day together organising a small version of the Socrates conference. We decide about the topics, we can spend the whole day coding or discuss the topics important for the company. 
+
+
+![Robert]({{ site.url }}/assets/img/custom/blog/lifeatcodurance/robert-2.jpg)
+
+### What are the responsibilities of a Senior Craftsman?
+
+We don’t define the strict boundaries of being the senior craftsman, but I think the expectations are created by yourself. You define the goals which move you outside your comfort zone. 
+
+I moved outside my comfort zones when I had to develop my skills around consultancy and coaching. As the Senior Craftsman, my personal expectations of myself are to understand not just the technical challenges of our customers, but the wider business considerations and goals. It is very important skill which has the crucial meaning when the problem is not trivial or controversial. 
+
+<blockquote class="hero bq-text-lg"><p>"In my opinion, a Senior Craftsman should be able to help others to develop."</p></blockquote>
+
+In my opinion, a Senior Craftsman should be able to help others to develop. Sharing knowledge allows all of us to develop from each other. Even if you are quite knowledgeable in some area, there is always the space for improvement. Exchanging knowledge increase also our communication skills because you need to explain people the concepts behind the idea.
```

Comment:
"Exchanging knowledge also increases our communication skills because you need to explain to people the concepts behind the idea."